src/v3/src/OktaSignIn/index.ts

```diff
@@ -237,13 +255,13 @@ export default class OktaSignIn {
     if (EVENTS_LIST.includes(eventName)) {
       // trap third-party callback errors
       const origHandler = eventHandler;
-      registeredEventHandler = (...callbackArgs) => {
+      registeredEventHandler = ((ctx: EventContext, error: EventErrorContext) => {
         try {
-          origHandler.call(this, ...callbackArgs);
+          origHandler.call(this, ctx, error);
         } catch (err) {
           console.error(`[okta-signin-widget] "${eventName}" event handler error:`, err);
         }
-      };
+      }) as OktaWidgetEventHandler;
```

Comment:
Can you file a ticket to fix this casting? I think it's possible to avoid this but I see why you've done it. Will just take some time with typescript docs and a little bit of creativity but I don't want this one to block the PR. Once you make the jira, please link it in a comment above this type cast.

Reply:
Changed `OktaWidgetEventHandler` declaration to
```
export type OktaWidgetEventHandler = EventCallback | EventCallbackWithError;
```